### ISSUE TITLE
iOS 9+ Location Request does nothing

### ISSUE THREAD

**Issue by WeetA34** [NativeScript/nativescript-geolocation#6]
Hi

enableLocationRequest function does nothing when called on iOS 9+.
As i'm a NativeScript beginner, i've spent multiple hours to understand where the problem was.
I've found it and is located in the following js file:
`node_modules/nativescript-geolocation/nativescript-geolocation.ios.js`
The Os Version test doesn't permit to match iOS 9+.
It can be corrected by something like the following:
```
 function enableLocationRequest(always) {
-    if (platformModule.device.osVersion.indexOf("8") === 0) {
+    var intOsVersionMajor = parseInt(platformModule.device.osVersion.split('.')[0]);
+    if (intOsVersionMajor >= 8) {
```

Regards
Stéphane

**Comment by Brainvise** [comment]
I think this is the culprit: https://github.com/NativeScript/nativescript-geolocation/blob/master/source/nativescript-geolocation.ios.ts#L169 which is only checking for version 8 and not version 9. If you change that to also check for 9 or check for > 8, then you should be good.

**Comment by WeetA34** [comment]
Hi Brainvise,

hummmm that's exactly what i explained in my comment :)
thx to confirm

Rgds
Stéphane

**Comment by Brainvise** [comment]
Oh sorry. Need to take the time to read.

**Comment by WeetA34** [comment]
no pb :)

**Comment by nsndeck** [comment]
Closed with commit #9

**Comment by vsinaswee** [comment]
Can't seem to get iOS 10 to ask for location permission. Android asks on location function run.
